src/hypertrace/agent/instrumentation/__init__.py

```diff
@@ -10,7 +10,18 @@
 # Setup logger name
 logger = logging.getLogger(__name__) # pylint: disable=C0103
 
+
+def is_json(body: str) -> bool:
+    """Checks if body is valid json"""
+    try:
+        json.loads(body)
```

Comment:
I am not 100% sure about this. Parsing the JSON body every time seems overkilling, can't we just do a best effort and check if it starts or ends with `[]` or `{}` and add the truncation accordingly.